10/umbraco-forms/developer/rendering-forms.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Learn the different ways of rendering a form on your website when using Umbraco Forms.
+---
+
+# Rendering Forms
+
+There are three options available for rendering a form.
+
+## Rendering Using a View Component
+
+To display a form in your view, you can make a call to a view component:
+
+```csharp
+@await Component.InvokeAsync("RenderForm", new { formId = Guid.Parse("<form guid>"), theme = "", includeScripts = true|false })
+```
+
+Four parameters can be provided:
+
+- `formId` is the GUID of a form.
+- `theme` is the name of a theme. If not provided, the default theme is used (see [Themes](./themes.md)).
+- `includeScripts` indicates whether scripts should be rendered with the form (see [Rendering Scripts](./rendering-scripts.md).
+- `recordId` is an optional existing record Id, used if editing records via the website is [enabled in configuration](../developer/configuration/README.md#alloweditableformsubmissions)
+
+Usually, rather than hard-coding the form's GUID, you'll use a form and/or theme picker property on your page:
+
+```csharp
+@await Component.InvokeAsync("RenderForm", new { formId = @Mode.Form, theme = @Model.Theme, includeScripts = false })
+```
+
+## Rendering Using a Tag Helper
+
+If you prefer a tag helper syntax, you can use one that ships with Umbraco Forms.
+
+Firstly, in your `_ViewImports.cshtml` file, add a reference to the Umbraco Forms tag helpers with:
+
+```csharp
+@addTagHelper *, Umbraco.Forms.Web
+```
+
+Then in your view you can use:
+
+```csharp
+@if (Model.Form is Guid formId)
+{
+    <umb-forms-render form-id="formId" theme="Model.Theme" include-scripts="false" />
+}
+```
```

Comment:
````suggestion
```cshtml
@if (Model.FormId is Guid formId)
{
    <umb-forms-render form-id="formId" theme="Model.FormTheme" exclude-scripts="true" />
}
```
````